101-create-win-vm-2nic-existingvnet/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Windows VM with 2 NIC in Existing VNET&Storage",
+  "description": " This template allows you to create Windows VM's  with 2 NICs with  existing Virtual network , existing storage account configuration ",
+  "summary": "This template allows you to create Windows VM's  with 2 NICs with  existing Virtual network , existing storage account configuration  ",
+  "githubUsername": "amalik99",
+  "dateUpdated": "<e.g. 2016-06-27>"
```

Comment:
this shouldn't be "<e.g. 2016-06-27>". It should be the actual date you submit on